task_DEV_1/task_DEV_1/Program.cs

```diff
@@ -10,18 +10,35 @@ static void Main(string[] args)
             int firstNumber = int.Parse(Console.ReadLine());
             Console.WriteLine("Enter the second number");
             int lastNumber = int.Parse(Console.ReadLine());
+            int min = 0;
+            int max = 0;
 
-            for (int i = firstNumber; i <= lastNumber; i++)
+            if (firstNumber < lastNumber)
+
+            {
+                min = firstNumber;
+                max = lastNumber;
+            }
+            else if (firstNumber > lastNumber)
+            {
+                min = lastNumber;
+                max = firstNumber;
+            }
+            else
+            {
+                Console.WriteLine("These numbers are equal");
```

Comment:
Hmmm.. We could leave so... but... if we input 4 and 4 program could check that 4 is divided on 2.